src/Apps/Order2/Routes/Checkout/Order2CheckoutRoute.tsx

```diff
@@ -58,7 +60,8 @@ export const Order2CheckoutRoute: React.FC<Order2CheckoutRouteProps> = ({
             : "width=device-width, initial-scale=1, maximum-scale=5 viewport-fit=cover"
         }
       />
-      <GridColumns>
+      {isLoading && <Order2CheckoutLoadingSkeleton />}
+      <GridColumns style={{ display: isLoading ? "none" : "block" }}>
```

Comment:
the `display` prop does not work on GridColumns. Could wrap it in a box if that is important.